cli/cmd/summary.go

```diff
@@ -5,22 +5,33 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"time"
+
+	"github.com/cloudquery/cloudquery/cli/internal/specs/v0"
+	"github.com/cloudquery/cloudquery/cli/internal/transformer"
+	"github.com/cloudquery/plugin-pb-go/managedplugin"
+	"github.com/cloudquery/plugin-pb-go/pb/plugin/v3"
+	"github.com/cloudquery/plugin-sdk/v4/caser"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/thoas/go-funk"
 )
 
 type syncSummary struct {
-	CliVersion          string `json:"cli_version"`
-	DestinationErrors   uint64 `json:"destination_errors"`
-	DestinationName     string `json:"destination_name"`
-	DestinationPath     string `json:"destination_path"`
-	DestinationVersion  string `json:"destination_version"`
-	DestinationWarnings uint64 `json:"destination_warnings"`
-	Resources           uint64 `json:"resources"`
-	SourceErrors        uint64 `json:"source_errors"`
-	SourceName          string `json:"source_name"`
-	SourcePath          string `json:"source_path"`
-	SourceVersion       string `json:"source_version"`
-	SourceWarnings      uint64 `json:"source_warnings"`
-	SyncID              string `json:"sync_id"`
+	CliVersion          string    `json:"cli_version"`
```

Comment:
```suggestion
	CLIVersion          string    `json:"cli_version"`
```

Reply:
Nevermind, I see this is because we use an automated mapping from field name to column name